airbyte-webapp/scripts/license-check.js

```diff
@@ -56,27 +56,28 @@ const IGNORED_PACKAGES = [`airbyte-webapp@${version}`];
  * Overwrite licenses for specific packages manually, e.g. because they can't be detected properly.
  */
 const LICENSE_OVERWRITES = {
-  'glob-to-regexp@0.3.0': 'BSD-3-Clause',
-  'rework@1.0.1': 'MIT',
```

Comment:
:information_source: Removed license overwrite for `rework` since no longer needed.